### DISCUSSION TITLE
Can't generate new secret keys: "Error: Secret Already Exists"

### DISCUSSION THREAD

**Discussion by Ethan-Guest** [swellstores/community#314] [Help] [answered]
I am trying to generate a new secret key to use with the backend API. In the dashboard, at **developer < API keys < Secret Keys**, when I click create key I get this error:

Secret Already Exists
{
  "field": "secret",
  "fields": {
    "secret": "true"
  }
}
![secret_key](https://github.com/swellstores/community/assets/25216738/dd1bc73f-a84c-4e1b-9888-2720354bc1a0)

I do not already have a secret key. Only a public key. Is it because I am in a test environment? Do I need to purchase a plan even during development? 

Thanks

**Comment by steven-k1** [comment]
Hey @Ethan-Guest, Thanks for posting about this. 

This is a known issue, and our team is actively working on it.

I'll get back to you when we have a fix,

**Comment by steven-k1** [comment, marked as answer]
@Ethan-Guest A fix was applied, and you should be good to generate keys as expected now.

If you continue to experience any issues, though, please let me know.

**Reply by Ethan-Guest** [reply]
Works for me, thanks for fixing this so quickly.

**Comment by morrisondavid** [comment]
This appears to be broken again. Both buttons are disabled

**Reply by ericingram** [reply]
Works for me on a quick test, can you share a screenshot of the issue you're seeing?
